### ISSUE TITLE
model.save and model.load about epoch numbers

### ISSUE THREAD

**Issue by ro6842** [tflearn/tflearn#529]
I have used the model.save to save pre-trained model, and then I use the model.load so as to retrain it.
But when I am training, the epoch number becomes "one", not the epoch number of pre-trained.

e.x I train the model until epoch number is 10(and the training step says 1000), and I save it. When I load this model and retrain, the epoch number becomes 0(but the training step re-starts at 1001), is this reasonable?

**Comment by aymericdamien** [comment]
The reason is that you may use new data to retrain your model, so epoch does not necessary have the same meaning for your new task. However steps records the number of gradient updates that has been performed and is saved.
We could also reset step to 0, but I am not sure what is the best way here, as both have their own pro and cons.

**Comment by ro6842** [comment]
Thanks, I got it!
